common/source/docs/common-mamba405-mk2.rst

```diff
@@ -0,0 +1,112 @@
+================================
+Mamba F405 MK2 Flight Controller
+================================
+
+.. image:: ../../../images/mamba_f405mk2_board.jpg
+     :target: ../_images/mamba_f405mk2_board.jpg
+
+https://www.diatone.us/products/mamba-f405-flight-controller-mk2
+
+The Mamba F405 MK2 is a flight controller produced by
+`Diatone <https://www.diatone.us>`__.
+
+.. note::
+
+   Support for this board is available with Plane 4.1.0, and higher
+
+Features
+========
+
+-  STM32F405RGT6 microcontroller
+-  MPU6000 IMU
+-  AT7456E OSD
+-  4 UARTs
+-  4 PWM outputs
+
+Pinout
+======
+
+.. figure:: ../../../images/mamba_f405mk2_pinout.png
+   :target: ../_images/mamba_f405mk2_pinout.png
+
+   Mamba F405 MK2 Board
+
+UART Mapping
+============
+
+======= ================ ================
+Name    Pin              Function
+======= ================ ================
+SERIAL0 COMPUTER         USB
+SERIAL1 PPM/RX1/SBUS/TX1 UART1 (RC Input)
+SERIAL3 TX3/RX3          UART3 (Telem1)
+SERIAL6 TX6/RX6          UART6 (GPS)
+======= ================ ================
+
+RC Input
+========
+
+RC input is configured on the PPM (UART1_RX) pin. It supports all RC
+protocols. The S.BUS pad provides an inverted input for S.BUS receivers.
+
+OSD Support
+===========
+
+The Mamba F405 MK2 supports OSD using OSD_TYPE 1 (MAX7456 driver).
+
+PWM Output
+==========
+
+The Mamba F405 MK2 supports up to 4 PWM outputs. The pads for motor
+output ESC1 to ESC4 on the above diagram are for the 4 outputs. All 4
+outputs support DShot as well as all PWM types.
+
+The PWM are in one group.
+
+Channels within the same group need to use the same output rate. If any
+channel in a group uses DShot then all channels in the group need to use
+DShot.
+
+Battery Monitoring
+==================
+
+The board has a builting voltage sensor. The voltage sensor can handle
+up to 6S LiPo batteries.
+
+The correct battery setting parameters are:
+
+-  BATT_MONITOR 3
+-  BATT_VOLT_PIN 11
+-  BATT_VOLT_MULT around 12.0
+-  BATT_CURR_PIN 13
+-  BATT_CURR_MULT around 39 with the included 40A ESC
+
+Compass
+=======
+
+The Mamba F405 MK2 does not have a builting compass, but you can attach
+an external compass using I2C on the SDA and SCL pads.
+
+Alternate settings
+==================
+
+It is possible to set alternate configurations with the BRD_ALT_CONFIG
+parameter.
+
+============== ============
+BRD_ALT_CONFIG RX1 function
```

Comment:
since there is no RX1 shown in the diagrams, it should be called PPM pin function in the heading.. and instead of referring to TX1 in the table , put a note that TX1 is usable in both configurations below it.

Also, we need to verify that ALT1 works, again due to controlled inverter output on PPM (RX1 pin)

Reply:
I pushed a commit that may address your feedback. Let me know how you like it.